CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/report/compare/release-3.1.1...master
 
+- Moved the legend on the distribution chart to the side
```

Comment:
Since this doesn't affect our API, we can make it a transparent change and use the `### Fixed` change set header

Reply:
Done